tests/schemas/invalid/test_invalid_answer_source_reference.json

```diff
@@ -0,0 +1,239 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "survey_id": "139",
+  "theme": "default",
+  "title": "Confirmation Question Test",
+  "data_version": "0.0.3",
+  "description": "A questionnaire to test answers referenced as source before it has been added.",
+  "navigation": {
+    "visible": true
+  },
+  "metadata": [
+    {
+      "name": "user_id",
+      "type": "string"
+    },
+    {
+      "name": "period_id",
+      "type": "string"
+    },
+    {
+      "name": "ru_name",
+      "type": "string"
+    },
+    {
+      "name": "trad_as",
+      "type": "string",
+      "optional": true
+    }
+  ],
+  "questionnaire_flow": {
+    "type": "Linear",
+    "options": {
+      "summary": {
+        "collapsible": false
+      }
+    }
+  },
+  "sections": [
+    {
+      "id": "confirmation-section",
+      "title": "Questions",
+      "groups": [
+        {
+          "id": "confirmation-block",
```

Comment:
How come we output the errors at the group level rather than the block with the reference? Also confusing that the group is called `confirmation-block`, particularly in the tests

Reply:
That's because  I've checked and the group can have skip conditions (as usual, can be based on answers) so I wanted that to be included in a schema as a theoretical example.

Reply:
It's something I overlooked, now they are properly displaying block, group or section id depending what level object we validate.

Reply:
I think the change has fixed this now

Reply:
it outputs section, group or block depending on where the validated object is.